core/src/main/scala/com/cloudera/livy/sessions/SessionManager.scala

```diff
@@ -41,6 +42,14 @@ class SessionManager[S <: Session, R](livyConf: LivyConf, factory: SessionFactor
   private[this] final val sessionTimeout =
     TimeUnit.MILLISECONDS.toNanos(livyConf.getLong(SessionManager.SESSION_TIMEOUT, 1000 * 60 * 60))
 
+  val livyHome = livyConf.livyHome().getOrElse {
+    if (livyConf.getBoolean("test", false)) {
```

Comment:
I'd rather have an env variable or system property set by the root pom instead of having to manually set this in the test code.